### ISSUE TITLE
Rogue Referrals

### ISSUE THREAD

**Issue by beamercola** [flpvsk/gatsby-plugin-gtag#24]
Is there a way to configure this plugin to work with utm/gclids?

Reference:
https://www.simoahava.com/gtm-tips/fix-rogue-referral-problem-single-page-sites/

**Comment by flpvsk** [comment]
I don't think so. Frankly I didn't understand the issue very well from the article, but the only config options the plugin passes to gtag are `page_path` and `anonymize_ip`. How exactly the `page_path` is constructed you can see here: https://github.com/flpvsk/gatsby-plugin-gtag/blob/master/src/gatsby-browser.js
